Hermes/appscale/hermes/__init__.py

```diff
@@ -204,7 +206,9 @@ def main():
   signal.signal(signal.SIGTERM, signal_handler)
   signal.signal(signal.SIGINT, signal_handler)
 
-  is_master = appscale_info.get_private_ip() == appscale_info.get_headnode_ip()
+  my_ip = appscale_info.get_private_ip()
+  is_master = my_ip == appscale_info.get_headnode_ip()
```

Comment:
Could you rename this variable to master_ip, because unlike the 'is_lb' below, this is not returning a boolean but the actual head node IP.